docs/manual/docs/cmd/spo/web/web-add.md

```diff
@@ -0,0 +1,64 @@
+# spo web add
+
+Create subsite
+
+## Usage
+
+```sh
+spo web add [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-t, --title [title]`|Subsite title
```

Comment:
title is a required option and should be denoted using angle brackets, like `<title>`